python/pyspark/__init__.py

```diff
@@ -48,6 +48,22 @@
 from pyspark.status import *
 from pyspark.profiler import Profiler, BasicProfiler
 
+
+def since(version):
+    """
+    A decorator that annotates a function to append the version of Spark the function was added.
+    """
+    import re
+    indent_p = re.compile(r'\n( +)')
+
+    def deco(f):
```

Comment:
Would it be OK to add a clause to handle situations where f.__doc__ is None? There are a handful of methods without docstrings. Thanks!